src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/ContainerExecDecorator.java

```diff
@@ -72,7 +70,6 @@ public ContainerExecDecorator(KubernetesClient client, String podName, String co
         this.client = client;
         this.podName = podName;
         this.containerName = containerName;
-        this.path = path;
```

Comment:
if path is removed, then the constructor should be deprecated and a new one without path argument added